portal/templates/psa_tracker.html

```diff
@@ -0,0 +1,77 @@
+{% extends "layout.html" %}
+{% block head %}
+	{{super()}}
+	 <link rel="stylesheet" href="{{ url_for('static', filename='css/psaTracker.css') }}">
+{% endblock %}
+{% block main %}
+<main id="mainPsaApp">
+    <input type="hidden" id="psaTrackerUserId" value="{{user.id}}"/>
+    <section class="psa-tracker-title">
+        <h2 class="tnth-header heading">{{_("Track Your PSA")}}</h2>
+        {{_("Prostate-specific antigen, or PSA, is a protein produced by cells of the prostate gland.  The PSA test measures the level of PSA in a man's blood.  For this test, a blood sample is sent to a laboratory for analysis.  The results are reported as nanograms of PSA per milliliter (ng/mL) of blood.")}}
```

Comment:
Looks like there's a few extra spaces after the periods, not sure if that's intentional or not...